includes/classes/class-omise-image.php

```diff
@@ -8,11 +8,13 @@ class Omise_Image {
 		 *
 		 * @param string $file Image file name with extension such as image.jpg
 		 * @param string $alternate_text Alternate text for the image
+		 * @param int $width Width size for image with default value at 30
+		 * @param int $height Max-height size for image with default value at 30
 		 * @return string HTML &lt;img&gt; element
 		 */
-		public static function get_image( $file, $alternate_text ) {
-			$url = WC_HTTPS::force_https_url( plugin_dir_url( '' ) . 'omise/assets/images/' );
-			return "<img src='$url/$file' class='Omise-Image' style='width: 30px; max-height: 30px;' alt='$alternate_text' />";
+		public static function get_image( $file, $alternate_text, $width = 30, $height = 30 ) {
+			$url = WC_HTTPS::force_https_url( plugin_dir_url( '' ) . 'omise/assets/images' );
+			return "<img src='$url/$file' class='Omise-Image' style='width: {$width}px; max-height: {$height}px;' alt='$alternate_text' />";
 		}
 	}
```

Comment:
Hey, I have a suggestion. We now can pass more than 2 arguments to the `get_image` function, the function call would look like this

```Omise_Image::get_image( 'file.jpg', 'Some text', 20, 25 );```

It would difficult to understand what the 20 and 25 represent. We have to go to the `get_image` function if we want to know more details. One way is to assign each values to a variables and pass that variable instead

```
$alternate_text = 'Some text';
$height = 60;
$width = 60;
Omise_Image::get_image( 'file.jpg', $alternate_text, $width, $height);
```

With the above way, we create more variables for readability purpose. In my view, It would be better if we change the arguments to a single array. 

```
Omise_Image::get_image([
    'file' => 'file.jpg',
    'alternate_text' => 'Text',
    'height' => 20,
    'width' => 25
]);
```

This would give clear meaning to the values and increase the readability.

What do you guys think of this?

Reply:
Yeah sure i agree here the update -> https://github.com/omise/omise-woocommerce/pull/275/commits/3efd81ea19f3aa080a400628ee4e6a34c5fec255 
nice suggestion 👍